### ISSUE TITLE
How to get train.dat and test.dat?

### ISSUE THREAD

**Issue by tyutjxs** [artem-oppermann/Deep-Autoencoders-For-Collaborative-Filtering#5]
Hi
   I try to study your demo, but my result is bad, so I guess there is a problem with my data.
   so How did you generate training sets and test sets from raw data?

**Comment by tyutjxs** [comment]
emmmm...

I have solved this problem, thank you very much for your code.

good luck！

**Comment by kalashnlkov** [comment]
@tyutjxs 
Could you please share your solution?

I have a overfitting result ( nice train_loss but increasing test_loss).

BTW, I randomize the raw data and split it with 9:1 to get train:test set.

Thank you in advance :)

**Comment by LucRyan** [comment]
@kalashnlkov 
Please check out this solved issue https://github.com/artem-oppermann/Deep-Autoencoders-For-Collaborative-Filtering/issues/2, I think you're having the same problem.
You have to train/test with similar distribution, otherwise you'll see it overfitting at the beginning.

**Comment by kalashnlkov** [comment]
@LucRyan 
Thanks for your reply. I've tried the approach #2 mentioned before but still get overfitting. 
[split.py](https://github.com/artem-oppermann/Restricted-Boltzmann-Machine/blob/master/data/train_test_split.py)
It's this possible sharing your train/test.dat?

**Comment by tyutjxs** [comment]
> @LucRyan
> Thanks for your reply. I've tried the approach #2 mentioned before but still get overfitting.
> [split.py](https://github.com/artem-oppermann/Restricted-Boltzmann-Machine/blob/master/data/train_test_split.py)
> It's this possible sharing your train/test.dat?

**Comment by tyutjxs** [comment]
> @tyutjxs
> Could you please share your solution?
> 
> I have a overfitting result ( nice train_loss but increasing test_loss).
> 
> BTW, I randomize the raw data and split it with 9:1 to get train:test set.
> 
> Thank you in advance :)

maybe you can reference my responsity https://github.com/tyutjxs/AutoEncoder-for-recommend-system,good luck !
